hazelcast/src/main/java/com/hazelcast/map/impl/operation/EntryBackupOperation.java

```diff
@@ -123,14 +114,10 @@ private boolean entryRemovedBackup(Map.Entry entry) {
         return false;
     }
 
-    private boolean entryAddedOrUpdatedBackup(Map.Entry entry) {
-        final Object value = entry.getValue();
-        if (value != null) {
```

Comment:
`entryRemovedBackup` method is handling `null` value, redundant to check it here I guess